UI/message.ts

```diff
@@ -10,6 +10,9 @@ export function* parseContent(content: string) {
     // npubs
     yield* match(/(nostr:)?npub[0-9a-z]{59}/g, content, "npub");
 
+    //nprofile
+    yield* match(/(nostr:)?nprofile[^ ]+/g, content, "nprofile");
```

Comment:
<img width="236" alt="image" src="https://github.com/BlowaterNostr/blowater/assets/127284497/38426456-dd00-41c0-aa87-1700ca9f5019">
This is apparently wrong and your unit tests failed to catch it

Reply:
What you need to match is characters that are part of bech32 encoding instead of "not space"